### PR TITLE
Feature/208 temporary term display

### PR DESCRIPTION
**Pull request by lukefisklennon**

### CODE REVIEW

File: client/src/components/Navbar.tsx [hunk 1]
```diff
@@ -49,6 +49,11 @@ const NavButton = styled(Button)`
   margin-right: 20px;
 `;
 
+const Weak = styled.span`
+  font-weight: 300;
+  opacity: 0.8;
```
Comment (esyw) [inline client/src/components/Navbar.tsx:54]: What do you think of adding left padding to match the gap between the logo and "Notangles", and/or decreasing the font to be a bit smaller than "Notangles"?

File: client/src/App.tsx [hunk 2]
```diff
@@ -276,11 +286,15 @@ const App: FunctionComponent = () => {
             isDarkMode={isDarkMode}
             handleDrawerOpen={handleDrawerOpen}
           />
-          <FriendsDrawer
-            isFriendsListOpen={isFriendsListOpen}
-            isLoggedIn={isLoggedIn}
-            setIsLoggedIn={handleSetIsLoggedIn}
-          />
+          {
+            IS_PREVIEW && (
```
Comment (esyw) [inline client/src/App.tsx:290]: I might've missed it but should menu button in the navbar be hidden as well?

Comment (lukefisklennon) [reply client/src/App.tsx:290]: Ah good point

File: client/src/components/Navbar.tsx (hunk 1 above)
Comment (lukefisklennon) [reply client/src/components/Navbar.tsx:54]: Yep ok, what do you think now?

Comment (esyw) [reply client/src/components/Navbar.tsx:54]: Might not even need the brackets with the difference in font to make it clear, but yeah looks good!

**Review by esyw — APPROVED** [review]

File: client/src/components/Navbar.tsx (hunk 1 above)
Comment (maxwowo) [reply client/src/components/Navbar.tsx:54]: Yeah I think it's just a bit cleaner without the brackets, not a big deal though

image
image

**Review by maxwowo — APPROVED** [review]

File: client/src/components/Navbar.tsx (hunk 1 above)
Comment (lukefisklennon) [reply client/src/components/Navbar.tsx:54]: Agreed, done now

**Pull request merged**